dask/dataframe/tests/test_groupby.py

```diff
@@ -42,8 +42,23 @@ def agg_func(request):
     return request.param
 
 
-def groupby_internal_repr():
-    pdf = pd.DataFrame({"x": [1, 2, 3, 4, 6, 7, 8, 9, 10], "y": list("abcbabbcda")})
+@pytest.mark.xfail(reason="uncertain how to handle. See issue #3481.")
```

Comment:
Fine to leave this as an xfail for now.

The difference is that pandas `GroupBy.__getitem__` will update `.obj` when we slice a single column. So `.obj` is a pandas Series.

In Dask, we apparently don't do that. We still have a dask DataFrame. Probably not a big deal.

Reply:
See above.